documentation/assemblies/deploying/assembly-kraft-mode.adoc

```diff
@@ -50,11 +50,42 @@ In the following example, three nodes perform a dual role and two nodes act only
 .Example cluster with dual-role nodes and dedicated broker nodes
 image::kraft-dual-role-quorum.png[KRaft cluster with nodes that combine roles]
 
-== KRaft limitations
+== Controller scaling
 
-Currently, the KRaft mode in Strimzi has the following major limitations:
+KRaft mode supports two types of controller quorums:
 
-* Scaling of KRaft controller nodes up or down is not supported.
+* *Static controller quorums* +
+In this mode, the number of controllers is fixed, and scaling requires downtime.
+* *Dynamic controller quorums* +
+This mode enables dynamic scaling of controllers without downtime. 
+New controllers join as observers, replicate the metadata log, and eventually become eligible to join the quorum.
+If a controller being removed is the active controller, it will step down from the quorum only after the new quorum is confirmed. 
+
+Scaling is useful not only for adding or removing controllers, but supports the following operations:
+
+* Replacing controllers because of hardware failure
+* Migrating clusters onto new hardware
+* Moving nodes from dedicated controller roles to dual roles or vice versa
```

Comment:
I don't think this really applies. On Kubernetes, you do the first two points by rescheduling the Pods. The 3rd point can also be done by adding or removing the broker roles.

On Kubernetes, the example of what might not be an obvious scaling operation is for example:
* Renaming a node pool (add a new node pool with new name and delete the old node pool with the old name)
* Changing non-JBOD storage (Create new node pool with new storage configuration and delete the old one)

The points you list here are more relevant to VMs / bare-metal Kafka.